apps/test-suite/tests/Image.js

```diff
@@ -57,6 +61,18 @@ export async function test(t, { setPortalChild, cleanupPortal }) {
           }
         }
       });
+
+      if (Platform.OS === 'ios') {
+        t.it('load animated image and emits animated is true', async () => {
+          const event = await mountAndWaitFor(
+            <Image source={ANIMATED_IMAGE_SOURCE} style={{ height: 100, width: 100 }} />,
+            'onLoad',
+            setPortalChild
+          );
+
+          t.expect(event.source.animated).toBe(true);
```

Comment:
Just one more thing 😂 
```suggestion
          t.expect(event.source.isAnimated).toBe(true);
```